src/web/frontend/components/UserProvider/UserProvider.tsx

```diff
@@ -1,54 +1,59 @@
 import React, { createContext, useState, useEffect } from "react";
 import jwt_decode from 'jwt-decode';
 import { DixitUser } from "../../../backend/entities/User";
+import { Player } from "../../../backend/entities/Player";
 import { JwtPayload } from '../../../backend/authentication/helpers';
 import axios from "axios";
+import cookie from "cookie";
 
 type UserData = {
   authenticated: boolean,
-  nickname: null | DixitUser['nickname'],
-  user_id: null | DixitUser['user_id'],
-  profile_picture: null | DixitUser['profile_picture']
+  nickname: DixitUser['nickname'],
+  user_id?: DixitUser['user_id'],
+  player_id?: Player['player_id'],
+  profile_picture: DixitUser['profile_picture']
 }
 
 type ContextData = {
-  user: null | UserData
+  user: UserData
+}
+
+const dummyUser = {
+  authenticated: false,
+  nickname: 'Guest',
+  profile_picture: 'anonymous_user.png'
 }
 
 const contextProps: ContextData = {
-  user: null
+  user: dummyUser
 }
 
 const context = createContext(contextProps);
 
 const UserProvider = ({ children }: any) => {
-  const [user, setUser] = useState<UserData>({
-      authenticated: false,
-      nickname: null,
-      user_id: null,
-      profile_picture: null
-  });
+  const [user, setUser] = useState<UserData>(dummyUser);
 
   useEffect(() => {
-    const token = localStorage.getItem('jwt_token');
+    let cookies = cookie.parse(document.cookie);
+    
+    if(cookies.oauth_jwt_token) {
+      localStorage.setItem('jwt_token', cookies.oauth_jwt_token);
+      document.cookie = "oauth_jwt_token=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;";
+    }
+
+    let token = localStorage.getItem('jwt_token');
 
-    axios.post('/api/auth/isAuthenticated', null, { headers: {"Authorization" : `Bearer ${token}`} })
+    axios.post('/api/auth/getUser', null, { headers: {"Authorization" : `Bearer ${token}`} })
```

Comment:
should be `/api/auth/user` or something, not get user

Reply:
ok, thanks. updated